smtp.c

```diff
@@ -14,8 +17,10 @@ static void get_random_bytes(void *buf, size_t n)
 {
 #if defined HAVE_ARC4RANDOM  /* BSD */
 	arc4random_buf(buf, n);
-#elif defined HAVE_GETRANDOM /* Linux */
+#elif defined HAVE_GETRANDOM /* Linux, GLIBC >= 2.25 */
 	getrandom(buf, n, 0);
+#elif defined HAVE_GETRANDOM_SYSCALL
+    syscall(SYS_getrandom, buf, n, 0);
```

Comment:
Nice, that does look pretty clean. :)

One small thing -- can you use a tab to indent here rather than spaces?

Reply:
Ah right, sorry, I'm more of a space person myself. :)
I'll fix that.